sync/src/tasks/tests.rs

```diff
@@ -782,7 +782,7 @@ async fn block_sync_task_test(total_blocks: u64, ancestor_number: u64) -> Result
     .generate();
     let result = sync_task.await?;
     assert!(!result.is_empty(), "task result is empty.");
-    let last_block_number = result
+    let last_block = result
```

Comment:
using last_block it is easy to get confused struct Block https://github.com/starcoinorg/starcoin/blob/v1.10.1/types/src/block.rs#L651
```
pub struct Block {
    /// The header of this block.
    pub header: BlockHeader,
    /// The body of this block.
    pub body: BlockBody,
}
```

Reply:
It's the same as above. The confusing statement here is far-fetched.

Reply:
OK, some other can review code。This is only my view.